test/integration/targets/dms_subnet_group/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+unsupported
+unstable
```

Comment:
New tests should not be marked `unstable`. Also, the tests should only be marked `unsupported` if we cannot add permissions to our CI system to run the tests or the tests will be too slow to be used in CI.